### DISCUSSION TITLE
"requirements not met" I followed the installation guide, please help me.

### DISCUSSION THREAD

**Discussion by hyp3else** [Atlas-OS/Atlas#1286] [Help & Support] [answered]
I followed the installation guide, please help me. 
Windows 11 Pro (24H2)
AMD Ryzen 5 5600
MSI B450M PRO M2 MAX
NVIDIA GEFORCE 1650
![image](https://github.com/user-attachments/assets/8180d715-c276-4e14-99e5-3f21090a4873)
if u can help me, write in discord - focuso666

**Comment by kamild1996** [comment, marked as answer]
If you want help on Discord, feel free to join the official Discord server: https://discord.com/servers/atlas-795710270000332800

If you installed 24H2 and you're seeing this message, it could only be because you're on an insider build. Post a screenshot from `winver` or from the "Windows specification" section of the Windows 11 settings (System > About).

**Reply by hyp3else** [reply]
nevermind, I somehow installed 23H2,but pressed to button "intsall windows 11 24h2"
